cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -1388,16 +1388,16 @@ pKeyOutputFormat =
     , Opt.value KeyOutputFormatTextEnvelope
     ]
 
-pPoolIdOutputFormat :: Parser PoolIdOutputFormat
+pPoolIdOutputFormat :: Parser IdOutputFormat
```

Comment:
The parser retains the `Pool` prefix however because the help text is pool specific.

Reply:
`IdOutputFormat` in isolation doesn't tell me anything. This obscures the type.